lib/utils.pm

```diff
@@ -406,6 +407,20 @@ sub workaround_type_encrypted_passphrase {
     unlock_if_encrypted;
 }
 
+# Handle the case when user is not selected, on gnome
+sub handle_user_not_selected {
+    my ($myuser) = @_;
+    $myuser //= $username;
+    assert_screen [qw(displaymanager-user-selected displaymanager-user-notselected)];
+    if (match_has_tag('displaymanager-user-notselected')) {
+        assert_and_click "displaymanager-$myuser";
+        record_soft_failure 'bsc#1086425- user account not selected by default, have to use mouse to login';
+    }
+    elsif (match_has_tag('displaymanager-user-selected')) {
```

Comment:
I am confused - function name saying that you handle not selected user, but you using needle with name "user-selected"

Reply:
I would recommend rename a function to `select_user_gnome`

Reply:
@asmorodskyi Done in latest version